docs/blueprints/SLRT.md

```diff
@@ -0,0 +1,42 @@
+Secure Launch Resource Table
+============================
+
+## Summary
+
+SLRT is a data format whose purpose is to describe Late Launch related
+information and pass it from a bootloader to DCE and then to DLME.  As SLRT is
+being processed it can even be updated in place.
+
+The format is defined in Secure Launch Specification.
```

Comment:
Please add a link to it.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/97941c079ed7b202e52b9bc265febac7b8afd04f..1a18d56ac6cad3dcb05e6dddf5ba68cdc918abfd#diff-6f6dfea2e585e2672fbe8db6b2ada95c775b48751cc04f75150235d6379aed8e